File: Java/TurnCount.java
/**
 * ターン数を表すクラスです。
 */
class TurnCount {
  private StandardIO standardIO = StandardIO.getInstance(); // 出力処理を担当するオブジェクトです。
  private int turnCount; // ターン数です。
  TurnCount() {
    turnCount = 0;
  }

  /**
   * ターン数のカウントを進めるメソッドです。
   */
  void countUp() {
    this.turnCount += 1;
  }

  /**
   * 現在のターン数をメッセージとして出力するメソッドです。
   */
  void printTurnCountMessage() {
    standardIO.println("【第" + this.turnCount + "ターン】");
  }
}

File: Kotlin/TurnCount.kt
/**
 * ターン数を表すクラスです
 * 
 * @constructor 何も行いません
 * 
 * @property turnCount ターン数です
 */
class TurnCount() {
    private var turnCount = 0

    /**
     * ターン数のカウントを進めるメソッドです
     */
    fun countUp() {
        this.turnCount += 1
    }

    /**
     * 現在のターン数を表すメッセージを作成・出力するメソッドです
     */
    fun printTurnCount() {
        StandardIO.println("【第 $turnCount ターン】")
    }
}